lib/ruby_lsp/ruby_lsp_rails/runner_client.rb

```diff
@@ -7,24 +7,29 @@
 module RubyLsp
   module Rails
     class RunnerClient
+      @instance = T.let(nil, T.nilable(RunnerClient))
+
```

Comment:
I'm not sure we want to make this a singleton. Other add-ons should only access the client through the `Addon#rails_runner_client` API to ensure proper ordering.